internal/core/ports/tx_builder.go

```diff
@@ -74,9 +60,9 @@ type TxBuilder interface {
 	VerifyForfeitTxs(
 		vtxos []domain.Vtxo, connectors tree.FlatTxTree, txs []string,
 	) (valid map[domain.Outpoint]ValidForfeitTx, err error)
-	BuildSweepTx(inputs []SweepableOutput) (txid string, signedSweepTx string, err error)
+	BuildSweepTx(inputs []TxInput) (txid string, signedSweepTx string, err error)
 	GetSweepableBatchOutputs(vtxoTree *tree.TxTree) (
-		vtxoTreeExpiry *arklib.RelativeLocktime, batchOutputs SweepableOutput, err error,
+		vtxoTreeExpiry *arklib.RelativeLocktime, batchOutputs TxInput, err error,
```

Comment:
if the name is `batchOutputs` why we return only one (that should eventually be like follows)? 
```suggestion
		vtxoTreeExpiry *arklib.RelativeLocktime, batchOutputs *TxInput, err error,
```